sample-apps/02-otel-collector/kubernetes/01-configmap.yaml

```diff
@@ -5,20 +5,55 @@ metadata:
   namespace: otel-collector
 data:
   otel-collector-config.yml: |
+    extensions:
+      sigv4auth:
+        region: "__replace-region__"
+        service: "osis"
+
     receivers:
       otlp:
         protocols:
           grpc:
             endpoint: 0.0.0.0:55680
+          http:
+            endpoint: 0.0.0.0:55681
             
+    processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 80
+        spike_limit_percentage: 30
+      batch:
+        send_batch_size: 10
+        timeout: 5s
+    
     exporters:
-      otlp/data-prepper:
-        endpoint: data-prepper.data-prepper.svc.cluster.local:21890
-        tls:
-          insecure: true
+      otlphttp/traces:
+        traces_endpoint: "https://__REPLACE WITH OtelTraces ADDRESS__/v1/traces"
+        auth:
+          authenticator: sigv4auth
+        compression: none
+      otlphttp/logs:
+        logs_endpoint: "https://__REPLACE WITH OtelLogs ADDRESS__/v1/logs"
```

Comment:
Change to:
`logs_endpoint: "https://__REPLACE_WITH_OtelLogs_ENDPOINT__/v1/logs"`

Reply:
template tags have been updated.

Issue to track TLS refactoring work: https://github.com/aws-samples/observability-with-amazon-opensearch/issues/62